extension/src/sidebar.css

```diff
@@ -1,6 +1,48 @@
-.promnesia body {
-    font-size: 15px;
-    color: black;
+@import url('https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;600&family=PT+Serif:ital,wght@0,400;0,700;1,400;1,700&display=swap');
+
+/* Ligt mode */
+.promnesia {
+    --opacity: 1;
+    --bg: rgba(255, 254, 252, var(--opacity));
+    --bg1: #e8eef0;
+    --bg2: #f3f4f6;
+    --bg3: #ddd;
+    --bg4: #ccc;
+    --color: #222;
```

Comment:
hmm, these names are really not descriptive, can we maybe add an explanation/link to the screenshot explaining what are these? 
(fine to do later once we settle on the final version)

Reply:
it's "2nd layer" of variables ))
most important selectors has own good named vars:
![image](https://user-images.githubusercontent.com/137919/166152017-714c29f3-50da-4127-8eaa-84172583c2f2.png)

but inside them i also using vars... THE THEME vars, their names here a little not speackable.
they reused in different properties, so can't be named `---border-color` as it's also used in shadow, etc...

it's like abstract vars, and then below more linked to blocks..

Reply:
hm, okay makes sense

Reply:
at first i  want to name them `ligh`, `lighter`, etc... but then i introduced dark mode and in became little confusing.
will try something like `primary`, `secondary`... or....
at least sinplify (again and again 😁) amount of used colors more...

Reply:
done!